.travis.yml

```diff
@@ -14,12 +14,12 @@ stages:
     if: tag =~ .*
 _install: &_install
   - sudo apt update -y
-  - sudo apt install -y libsnappy-dev
+  - sudo apt install -y libsnappy-dev gzip
   - travis_retry make bblfsh-start
   - pip install --upgrade pip cython codecov
-  - ML_MINING_SETUP_INCLUDE_TESTS=1  pip install ".[tf, pandas]"
+  - ML_MINING_SETUP_INCLUDE_TESTS=1 pip install ".[tf, pandas]"
   - cd $(pip show sourced.ml.mining|grep Location|cut -d' ' -f2)/sourced/ml/mining
-  - find . -wholename "*/tests/*" -type d -exec chmod 555 {} \;
```

Comment:
you should keep this line.
We should not tough test directory while testing.

Reply:
but the tsts uncrompresses tar files. If I freeze this I get errors

Reply:
but you can read it and decompress to a temporary location.

Reply:
I tried, but I don't know how to make the decompression process, which is called using `subprocess.check_call(["gzip", "-fdk", path + ".gz"])` to write to a temporary file.

Do you know how to redirect the target of a subprocess call? otherwis I will need to rewrite all the tests, and I am not sure it is worth it...

Reply:
Solved